libsplinter/src/network/dispatch.rs

```diff
@@ -602,6 +618,37 @@ where
     }
 }
 
+impl<MT, Source> DispatchLoop<MT, Source>
+where
+    MT: Any + Hash + Eq + Debug + Clone + Send,
+    Source: Send + 'static,
+{
+    pub fn builder() -> DispatchLoopBuilder<MT, Source> {
```

Comment:
This should probably be in a separate commit

Reply:
split out

Reply:
split out